hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java

```diff
@@ -281,10 +287,12 @@ private static OMClientRequest getOMAclRequest(OMRequest omRequest) {
       } else if (ObjectType.BUCKET == type) {
         return new OMBucketRemoveAclRequest(omRequest);
       } else if (ObjectType.KEY == type) {
-        if (isBucketFSOptimized()){
+        OMKeyRemoveAclRequest aclReq = new OMKeyRemoveAclRequest(omRequest);
+        OmBucketInfo buckInfo = aclReq.getBucketInfo(ozoneManager);
+        if (buckInfo.equals(BucketLayout.FILE_SYSTEM_OPTIMIZED)) {
```

Comment:
`buckInfo.equals(BucketLayout.FILE_SYSTEM_OPTIMIZED)`
There is a findbug on this statement.

Please use getLayout while comparing, spomething like `buckInfo.getLaytout().equals(BucketLayout.FILE_SYSTEM_OPTIMIZED)`

Reply:
Done